### PR TITLE
(BOLT-1243) Error with helpful message when no CLI command given

### PR DESCRIPTION
**Pull request by donoghuc**
Previously an missing command argument would result in a stack trace. This commit validates that a command is given when invoking `bolt command run` on the CLI is a string with at least one character. Note this validation is modeled after the plan function `run_command` which validates the command argument is of type `String[1]`.

### CODE REVIEW

File: spec/bolt/cli_spec.rb [hunk 1]
```diff
@@ -2108,7 +2122,7 @@ def stub_config(file_content = {})
       after(:each) { ENV.delete('BOLT_INVENTORY') }
 
       it 'errors when BOLT_INVENTORY is set' do
-        cli = Bolt::CLI.new(%W[command run --inventoryfile #{File.join(inventorydir, 'invalid.yml')} --nodes foo])
+        cli = Bolt::CLI.new(%W[command run id --inventoryfile #{File.join(inventorydir, 'invalid.yml')} --nodes foo])
```
Comment (lucywyman) [inline spec/bolt/cli_spec.rb]: Haha, one of these things is not like the others :)

File: spec/bolt/cli_spec.rb [hunk 2]
```diff
@@ -1989,31 +2003,31 @@ def stub_config(file_content = {})
...
+        cli = Bolt::CLI.new(%W[command run uptime --configfile #{conf.path} --nodes foo])
         cli.parse
         expect(cli.config.transports[:ssh]['host-key-check']).to eq(false)
       end
     end
 
     it 'reads run-as for ssh' do
       with_tempfile_containing('conf', YAML.dump(complete_config)) do |conf|
-        cli = Bolt::CLI.new(%W[command run --configfile #{conf.path} --nodes foo --password bar --no-host-key-check])
+        cli = Bolt::CLI.new(%W[command run r --configfile #{conf.path} --nodes foo --password bar --no-host-key-check])
```
Comment (lucywyman) [inline spec/bolt/cli_spec.rb]: Is this an actual command?

Comment (donoghuc) [reply spec/bolt/cli_spec.rb]: It doesn't need to be an actual command. The reason I used a single character is to avoid line length rubocop

Comment (lucywyman) [reply spec/bolt/cli_spec.rb]: Ah, makes sense :+1:

**Pull request merged**